pkg/abstract/typed_change_item.go

```diff
@@ -243,7 +229,7 @@ func packTypedValue(val any) (TypedValue, error) {
 			errs = util.AppendErr(errs, err)
 		}
 		if len(errs) > 0 {
-			return *new(TypedValue), xerrors.Errorf("many errs: %w", errs)
+			return TypedValue{}, xerrors.Errorf("many errs: %w", errs)
```

Comment:
all this construct:
`*new(SomeType)` instead of `SomeType{}` is a result of our policy of exhausted structs.
Struct should be either fully initiatiated (with all fields) or be initted like this.
We use fork of this linter: https://github.com/mbilski/exhaustivestruct, linters would be enabled in ci here: #177